internal/commands/logs/list_inputs_test.go

```diff
@@ -27,5 +29,24 @@ func TestLogTypes(t *testing.T) {
 
 			assert.Equal(t, tc.logTypes, i.logTypes())
 		})
+
+		t.Run("should fail if log type "+tc.logType+" is not formatted before passed to client", func(t *testing.T) {
+			realmClient := mock.RealmClient{}
+			realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+				return []realm.App{{}}, nil
+			}
+			realmClient.LogsFn = func(groupID, appID string, opts realm.LogsOptions) (realm.Logs, error) {
+				assert.Equal(t, tc.logTypes, opts.Types)
```

Comment:
it's generally a good idea to keep asserts out of mock function implementations like this, its a common practice to have a `var logsOpts realm.LogsOptions` outside of the scope of the `LogsFn`, and then assign to it inside the function implementation, so that it can be asserted on downstream in the test